templates/partials/button.html

```diff
@@ -0,0 +1,48 @@
+{% if med_action_button%}
+    {% set button_type = "button-med" %}
+    {% set button_background_colour = "bg-button-primary" %}
+    {% set button_font = "button-text" %}
+    {% set button_font_colour = "text-on-primary" %}
+{% endif %}
```

Comment:
if these if statements are not meant to overlap, use if/elif

Reply:
Changed to use elif.